templates/common-partials/script/filterbox.hbs

```diff
@@ -10,4 +10,4 @@ ANSWERS.addComponent("FilterBox",
   {{{json componentSettings.FilterBox}}}
 ]
 {{/shallowMergeConfig}}
-));
\ No newline at end of file
+);
```

Comment:
could we add a snapshot or something for this? maybe just add static filters to one of the vertical pages